frontends/web/src/routes/account/utils.ts

```diff
@@ -15,7 +15,21 @@
  * limitations under the License.
  */
 
-import { CoinCode, ScriptType } from '../../api/account';
+import { CoinCode, ScriptType, IAccount } from '../../api/account';
+
+export function getAccount(accounts: IAccount[], accountCode: string | undefined): IAccount | undefined {
+  if (!accounts) {
+    return undefined;
+  }
```

Comment:
accounts is alway at least an empty array `[]` and never falsy, `if (!accounts) {` can be removed.

Reply:
Done, thanks :)